test/signs/realtime_test.exs

```diff
@@ -8,6 +8,7 @@ defmodule Signs.RealtimeTest do
   @src %Signs.Utilities.SourceConfig{
     stop_id: "1",
     direction_id: 0,
+    routes: ["Red"],
```

Comment:
Is there a case where this is ever absent? Should we actively prevent it from being absent, using `@enforce_keys`?

Reply:
Good call, done.